src/management/cameras.php

```diff
@@ -0,0 +1,109 @@
+<?php
+$servername = "db";
+$username = "db_username";
+$password = "db_password";
+$dbname = "db_name";
+
+// Create connection
+$conn = new mysqli($servername, $username, $password, $dbname);
+
+// Check connection
+if ($conn->connect_error) {
+  die("Connection failed: " . $conn->connect_error);
+}
+
+// Fetch city name from the GET parameters
+$city = $_GET['city'];
+
+// Fetch cameras for the selected city
+$camerasQuery = "SELECT * FROM cameras
+                 INNER JOIN parking_spaces ON cameras.parking_space_id = parking_spaces.id
+                 WHERE parking_space.city = '$city'";
+$camerasResult = $conn->query($camerasQuery);
+
+// Close the database connection
+$conn->close();
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title><?php echo "Manage City - $city"; ?></title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      margin: 20px;
+    }
+
+    h1 {
+      color: #333;
+    }
+
+    /* Add condition to set background color if city is D�sseldorf */
+    <?php if ($city == 'D�sseldorf') : ?>h1 {
```

Comment:
It looks like the file is not using UTF-8 encoding format so it makes problems with non-Latin characters.
Also, I think it is good idea to move custom stylesheets to another file.